src/ui/public/directives/__tests__/fixed_scroll.js

```diff
@@ -9,10 +9,12 @@ import Promise from 'bluebird';
 describe('FixedScroll directive', function () {
 
   let compile;
+  let timeout;
```

Comment:
I've removed the timeout flushing and mocked `debounce` instead so that the tests don't need to know anything about debounce's implementation.